### DISCUSSION TITLE
Create new file

### DISCUSSION THREAD

**Discussion by WeebNetsu** [spacedriveapp/spacedrive#1984] [Feature Request]
Using this on MacOs, I would love if we could have a 'Create New File' option on right click

**Comment by utkubakir** [comment]
I assume it would just create a blank file and you would enter the extension right?

**Reply by WeebNetsu** [reply]
Yes! Just something sweet and simple

**Reply by utkubakir** [reply]
Awesome, we had this in our backlog, will move it higher on our todo-list! (https://github.com/spacedriveapp/spacedrive/issues/1989)

**Reply by WeebNetsu** [reply]
Thank you very much!
